### PR TITLE
Interpolated variables

### PR DESCRIPTION
**Pull request by kfinley-rbi**
Fixes Interpolated variables, e.g.

```
width: `${WIDTH_PX}px`,
```

### CODE REVIEW

File: examples/__testfixtures__/styled-components-to-ucl/interpolated.output.ts [hunk 1]
```diff
@@ -1,14 +1,18 @@
 import { Box } from '@rbilabs/universal-components';
 
-const MAX_CONTENT_WIDTH_PX_DESKTOP = 12;
+const WIDTH_PX = 12;
+const HEIGHT_PX = 12;
 
 export const InterpolatedVariables = Box.withConfig({
-  width: `${MAX_CONTENT_WIDTH_PX_DESKTOP}px`,
+  width: `${WIDTH_PX}px`,
+  height: `pre${HEIGHT_PX}rem`,
```
Comment (bkasten-rbi) [inline examples/__testfixtures__/styled-components-to-ucl/interpolated.output.ts:8]: Should we be concerned about this?

Why we should we allow the ‘pre’ in there? And then should we comment it out if it includes rem and require people to manually convert?

**Review by bkasten-rbi — APPROVED** [review]
Amazing. One question on allowing rems, but the approach is legit

File: examples/__testfixtures__/styled-components-to-ucl/interpolated.output.ts (hunk 1 above)
Comment (kfinley-rbi) [reply examples/__testfixtures__/styled-components-to-ucl/interpolated.output.ts:8]: `pre` is probably a bad example. I think we are doing thing like `-${HEIGHT_PX}px`, though

For `rem` yes, maybe we should just error on any `post` that isn't `px`?

Comment (bkasten-rbi) [reply examples/__testfixtures__/styled-components-to-ucl/interpolated.output.ts:8]: 👍

Comment (bkasten-rbi) [reply examples/__testfixtures__/styled-components-to-ucl/interpolated.output.ts:8]: I'm going to merge and we can iterate

**Pull request merged**